google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/DeleteCells.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.models;
+
+import com.google.auto.value.AutoValue;
+import com.google.cloud.bigtable.data.v2.models.Range.TimestampRange;
+import com.google.common.base.MoreObjects;
+import com.google.protobuf.ByteString;
+import java.io.Serializable;
+import javax.annotation.Nonnull;
+
+/** Representation of a DeleteCells mod in a data change. */
+@AutoValue
```

Comment:
Here and everywhere else: `@InternalApi`

Reply:
Are these InternalApi? The user will be able to access these mods through something like:

ChangeStreamRecord record = CdcBeamCode.readChangeStreamBlah();
if (record instanceof ChangeStreamMutation) {
  ChangeStreamMutation changeStreamMutation = (ChangeStreamMutation) record;
  for (Entry entry : changeStreamMutation.getEntries()) {
    if (entry instanceof DeleteCells) {
      DeleteCells deleteCells = (DeleteCells) entry;
      // Do whatever the users want to do with deleteCells.
      System.out.println(deleteCells.getFamilyName().......);
    }
  }
}